iot/pom.xml

```diff
@@ -19,6 +19,8 @@
     <!-- align with Hono compatible versions -->
     <spring.boot.version>2.1.5.RELEASE</spring.boot.version>
     <micrometer.version>1.1.4</micrometer.version>
+    <jaeger-client.version>0.34.0</jaeger-client.version>
+    <opentracing.version>0.31.0</opentracing.version>
```

Comment:
Should we move this to the topevel pom? Jus thinking we want to standardize on the same version across enmasse and we could get into the situation where we duplicate this info because we forgot this being here.

Reply:
I moved this to the top level.